config/routes.rb

```diff
@@ -6,6 +6,10 @@
 
   get "/", to: redirect("https://www.gov.uk/find-coronavirus-support")
 
+  if ActiveModel::Type::Boolean.new.cast(ENV["REROUTE_TO_LIVE"])
+    get "*everything", to: redirect("https://www.gov.uk/find-coronavirus-support")
+  end
```

Comment:
I don't think we need the flag - we'll just deploy redirect after the migration. 

We can also remove all the other routes and just have a single wildcard redirect.

Reply:
That will break all the tests which will mean the code won't get through CI.